app/views/public_pages/stimulus.html.erb

```diff
@@ -32,6 +32,16 @@
           <% t("views.public_pages.stimulus_payment.intro.eligibility.info").each do |paragraph| %>
             <p><%= paragraph %></p>
           <% end %>
+          <ul class="list--bulleted">
+            <% t("views.public_pages.stimulus_payment.intro.eligibility.list").each do |item| %>
+              <li>
+                <%= item %>
+              </li>
+            <% end %>
+          </ul>
```

Comment:
The new copy needed to list things. I took this from a different example from `public_pages/home.html.erb`